samples/TestEntityExpansionLimit/src/main/java/xxe/DemoWithSecurityManager.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package xxe;
```

Comment:
Can we use the standard package naming for this. This should be 

org.wso2.security.tools.samples.xee.jaxb
org.wso2.security.tools.samples.xee.dom

Reply:
Done

Reply:
Package name is still "xxe.dom" 

Also, this is entity expansion not "external entity injection (XXE)". Hence, let's invent a name (xee) or use "entityexpansion" in package.